_data/trip_fees.yml

```diff
@@ -39,9 +39,13 @@
     price: 10
     category: trip
 -
-    name: "MITOC Rock Program $100"
+    name: "School of Rock Fee"
     price: 100
     category: trip
+-
+    name: "School of Rock T-Shirt"
+    price: 40
+    category: trip
```

Comment:
I know that this isn't a trip, but I don't see any way to do a set fee for a gear rental/purchase.

Reply:
Gear purchases are in `_data/gear_prices.yml`. That list populates https://mitoc.mit.edu/rentals/prices

For example, a MITOC shirt:
https://github.com/mitoc/website/blob/4c69b6fbf0ab2c2fb12ff7be3145af94204b142e/_data/gear_prices.yml#L212-L213

Reply:
Yes, but that won't cause it to show up on the /pay page. We want to have it there to make it easy to track who has completed payment for their shirts.